phpBB/styles/prosilver/template/smiley_dropdown.html

```diff
@@ -0,0 +1,40 @@
+    <div id="smiley-dropdown" class="dropdown dropdown-extended" >
+        <div class="pointer"><div class="pointer-inner"></div></div>
+        <div class="smiley-content">
+            <div id="smiley-box" class="smiley-box">
+                <ul>
+                    <!-- EVENT posting_editor_smilies_before -->
+                    <!-- IF S_SMILIES_ALLOWED and .smiley -->
+                    <!-- BEGIN smiley -->
+                    <li class="smiley-list">
+                        <a href="#" onclick="insertText('{smiley.A_SMILEY_CODE}')"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
+                </li>
+                    <!-- END smiley -->
+                    <!-- ENDIF -->
+                    <!-- IF S_SHOW_SMILEY_LINK and S_SMILIES_ALLOWED -->
+                <br /><li><a href="{U_MORE_SMILIES}" onclick="popupSmileys(this.href)">{L_MORE_SMILIES}</a></li>
```

Comment:
@hanakin Do you have any idea about this. If soo could you help me with this. Sorry for bothering you. Nd no hurry.